### ISSUE TITLE
Doesn't configure with cmake 3.5

### ISSUE THREAD

**Issue by VorpalBlade** [FedeDP/libmodule#4]
```
$ cmake ..
CMake Error at CMakeLists.txt:3 (project):
  project with VERSION must use LANGUAGES before language names.


-- Configuring incomplete, errors occurred!
$ cmake --version
cmake version 3.5.1

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```
It looks like cmake 3.5.1 does not support the "DESCRIPTION" attribute to project():
```
$ cmake --help-command project
[...]
 project(<PROJECT-NAME> [LANGUAGES] [<language-name>...])
 project(<PROJECT-NAME>
         [VERSION <major>[.<minor>[.<patch>[.<tweak>]]]]
         [LANGUAGES <language-name>...])

[...]
```

I see two solutions: either change cmake_minimum_required (which will mean more work for me, since I will have to build a separate cmake) or change the file to be compatible with the currently specified minimum cmake version.

I tried adding "LANGUAGES C" in the middle as well as the end of that list, but that doesn't work, getting errors such as "Could not find cmake module file: CMakeDetermineDESCRIPTIONCompiler.cmake".

Removing the description does work however, I'm not sure if it can be added in some other way instead perhaps. It might be possible to do an if statement on cmake version and do two different project clauses as well.

**Comment by FedeDP** [comment]
What about using: 
`project(module VERSION 3.0.1 LANGUAGES C)`
?

**Comment by VorpalBlade** [comment]
Yes that works. But adding DESCRIPTION to at any point makes it fail.

**Comment by FedeDP** [comment]
I think is okay to avoid adding DESCRIPTION for me. I will push updated CMakeLists.

**Comment by FedeDP** [comment]
Fixed in e4b92f7f93116063d3e202d6568861b8b80f77eb.
